src/commands/nps/coverage/verify.ts

```diff
@@ -60,11 +56,12 @@ export default class Verify extends SfdxCommand {
 		return { success: true, allClassesWithCoverage };
 	}
 
-	get allClassesWithCoverage() {
+	private getClassesWithCoverage()  {
 		// Note: sometimes NaN happens because numLocations and numLocationsNotCovered are both 0,
 		// for example, in Constants classes with just one line for a single constant variable
+		const apexTestResults: ApexTestResults = this.apexTestResults();
 		const result = this.flags.classes
-			.map((className) => this.mapCalculatedCoverageToClassName(className))
+			.map((className) => this.calculateCoverage(className, apexTestResults))
```

Comment:
I would, instead, have this line directly as	 `.map((className) => this.calculateCoverage(apexTestResults[className]))` so we avoid passing as a parameter a large object reference, what do you think?

Reply:
Agree, good point